Sprint-1/interpret/time-format.js

```diff
@@ -12,13 +12,21 @@ console.log(result);
 // For the piece of code above, read the code and then answer the following questions
 
 // a) How many variable declarations are there in this program?
+// There are six variable declarations
 
 // b) How many function calls are there?
+// 1 function call
 
 // c) Using documentation, explain what the expression movieLength % 60 represents
+// % is a modulus operator. It divides movieLength by 60 and returns the remainder.
```

Comment:
This is a literal translation of the statement. Can you also explain in lament terms what this statement tries to calculate?